docs/source/06_nodes_and_pipelines/04_micro_packaging.md

```diff
@@ -0,0 +1,124 @@
+    ├── my_project
+    │   ├── __init__.py
+    │   └── pipelines
+    │       └── {{pipeline_name}}    <-- Pipeline folder
+    └── tests
+        ├── __init__.py
+        └── pipelines
+            └── {{pipeline_name}}    <-- Pipeline tests
+```
+
+Kedro will also include any requirements found in `src/<python_package>/pipelines/<pipeline_name>/requirements.txt` in the modular pipeline wheel file. These requirements will later be taken into account when pulling a pipeline via `kedro pipeline pull`.
+
+```eval_rst
+.. note::  Kedro will not package the catalog config files even if those are present in ``conf/<env>/catalog/<pipeline_name>.yml``.
+```
+
+If you plan to publish your packaged modular pipeline to some Python package repository like [PyPI](https://pypi.org/), you need to make sure that your modular pipeline name doesn't clash with any of the existing packages in that repository. However, there is no need to rename any of your source files if that is the case. Simply alias your package with a new name by running `kedro pipeline package --alias <new_package_name> <pipeline_name>`.
+
+In addition to [PyPI](https://pypi.org/), you can also share the packaged wheel file directly, or via a cloud storage such as AWS S3.
+
+## Package multiple modular pipelines
+
+To package multiple modular pipelines in bulk, run `kedro pipeline package --all`. This will package all pipelines specified in the `tool.kedro.pipeline.package` manifest section of the project's `pyproject.toml` file:
+
+```toml
+[tool.kedro.pipeline.package]
+first_pipeline = {alias = "aliased_pipeline", destination = "somewhere/else", env = "uat"}
+second_pipeline = {}
+```
+
+* The keys (`first_pipeline`, `second_pipeline`) are the names of the modular pipeline folders within the codebase.
+* The values are the options accepted by the  `kedro pipeline package <pipeline_name>` CLI command.
+
+```eval_rst
+.. note::  Make sure `destination` is specified as a POSIX path even when working on a Windows machine.
+```
+
+## Pull a modular pipeline
+
+You can pull a modular pipeline from a wheel file by executing `kedro pipeline pull <package_name>`.
+
+* The `<package_name>` must either be a package name on PyPI or a path to the wheel file.
+* Kedro will unpack the wheel file, and install the files in following locations in your Kedro project:
+  * All the modular pipeline code in `src/<python_package>/pipelines/<pipeline_name>/`
+  * Configuration files in `conf/<env>/parameters/<pipeline_name>.yml`, where `<env>` defaults to `base`.
+  * To place parameters from a different config environment, run `kedro pipeline pull <pipeline_name> --env <env_name>`
+  * Pipeline unit tests in `src/tests/pipelines/<pipeline_name>`
+* Kedro will also parse any requirements packaged with the modular pipeline and add them to project level `requirements.in`.
+* It is advised to do `kedro install --build-reqs` to compile and install the updated list of requirements after pulling a modular pipeline.
```

Comment:
We've updated the advice in other places already I believe, given it's deprecated functionality:
```suggestion
* It is advised to do `kedro build-reqs` to compile the updated list of requirements after pulling a modular pipeline.
```